src/main/java/frc/robot/subsystems/LimelightSubsystem.java

```diff
@@ -45,20 +53,51 @@ public void periodic() {
       SmartDashboard.putNumber("target Id", table.getEntry("tid").getDouble(0.0));
       SmartDashboard.putNumber("pipe line", pipeln.getDouble(0.0));
       SmartDashboard.putNumber("distance constantly updated", getDistance());
+      System.out.println(getDistance() + "sees a target!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!");
     }
     // TODO get tgt ID  - based of pipeline
     else {
       SmartDashboard.putBoolean("Target Seen", false);
       SmartDashboard.putNumber("target Id", -99);
       SmartDashboard.putNumber("pipe line", -99);
     }
+
+    // SmartDashboard.putNumber("april tag
+    // id",NetworkTableInstance.getDefault().getTable("limelight").getEntry("tid").getDouble(-10000));
+
+    SmartDashboard.putNumber("distance constantly updated", getDistance());
+
+    pipelineChooser.setDefaultOption(
+        "Red Tag 1", 0); // TODO add the other tags that are on the field
+    pipelineChooser.setDefaultOption("Red Tag 2", 1);
+    pipelineChooser.setDefaultOption("Red Tag 3", 2);
+    pipelineChooser.setDefaultOption("Red Loading Zone Tag 5", 5);
+
+    pipelineChooser.setDefaultOption("Blue Tag 6", 4);
+    pipelineChooser.setDefaultOption("Blue Tag 7", 6);
+    pipelineChooser.setDefaultOption("Blue Tag 8", 7);
+    pipelineChooser.setDefaultOption("Blue Loading Zone Tag 4", 5);
```

Comment:
Does the drive team need a reference diagram of where the tag numbers are?

Reply:
we can definitely print one out if they want one